src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private Map<String, Integer> storage = new HashMap<>();
+
+    public void addFruit(String fruit, int quantity) {
+        storage.put(fruit, storage.getOrDefault(fruit, 0) + quantity);
+    }
+
+    public void removeFruit(String fruit, int quantity) {
+        if (storage.containsKey(fruit)) {
+            storage.put(fruit, storage.get(fruit) - quantity);
+        }

```

Comment:
When implementing the removeFruit method, you should check that the resulting balance after the removal is not negative. If it is, you should throw a RuntimeException according to the checklist item 'Handling Purchase operation.'